server/src-lib/Hasura/GraphQL/Validate/Field.hs

```diff
@@ -144,7 +144,7 @@ withDirectives dirs act = do
       when (isJust $ _aivVariable val) markNotReusable
       case _aivValue val of
         AGScalar _ (Just (PGValBoolean v)) -> return v
-        _ -> throw500 "did not find boolean scalar for if argument"
+        _                                  -> throw500 "did not find boolean scalar for if argument"
```

Comment:
@nicuveo This kinda looks awkward. Is this a setting that's configurable? To not align if (longest - shortest) > some meaningful value?